pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterTest.java

```diff
@@ -786,6 +786,15 @@ public JsonNode cancelQuery(String clientQueryId)
     return null; // TODO
   }
 
+  /**
+   * Execute a query and extract the count result
+   */
+  protected int getTotalDocCountFromQuery(String query) throws Exception {
```

Comment:
`TotalDocCount` might be a little misleading since it sounds like the total number of docs in the table - maybe something like `getQueryNumResultRows` might be clearer?